Example Apps/Example ObjC/AlertManager.m

```diff
@@ -15,19 +15,23 @@
 @implementation AlertManager
 
 + (void)sendAlertWithManager:(SDLManager *)sdlManager image:(nullable NSString *)imageName textField1:(NSString *)textField1 textField2:(nullable NSString *)textField2 {
-    SDLSoftButton *okSoftButton = [[SDLSoftButton alloc] initWithType:SDLSoftButtonTypeText text:AlertOKButtonText image:nil highlighted:YES buttonId:10000 systemAction:nil handler:nil];
-    SDLAlert *alert = [[SDLAlert alloc] initWithAlertText1:textField1 alertText2:textField2 alertText3:nil softButtons:@[okSoftButton] playTone:YES ttsChunks:nil duration:5000 progressIndicator:NO alertIcon:nil cancelID:0];
+    SDLSoftButtonObject *okSoftButton = [[SDLSoftButtonObject alloc] initWithName:AlertOKButtonText text:AlertOKButtonText artwork:nil handler:nil];
+    SDLAlertView *alert = [[SDLAlertView alloc] initWithText:textField1 buttons:@[okSoftButton]];
```

Comment:
Setting this alert view timeout to 3 seconds doesn't work on manticore. That might be a manticore issue though?

Reply:
There is a currently uindoucmentated feature where the "duration" is ignored if a soft button is added to the alert (https://github.com/smartdevicelink/rpc_spec/issues/235). If you remove the soft button, the timeout works file. 

I added clarifying documentation to the `defaultTimeout` and `timeout` parameters to let developers know that those values will be ignored if they add a button to the alert.